packages/libs/user-datasets/src/lib/Components/List/UserDatasetList.tsx

```diff
@@ -49,12 +49,12 @@ interface Props {
   userDatasets: UserDataset[];
   filterByProject: boolean;
   shareUserDatasets: (
-    userDatasetIds: number[],
+    userDatasetIds: string[],
     recipientUserIds: number[]
   ) => any;
   unshareUserDatasets: (
-    userDatasetIds: number[],
-    recipientUserIds: number[]
+    userDatasetIds: string,
```

Comment:
For posterity since we already discussed this in another venue, our UI currently doesn't allow for us to batch together multiple unshare requests. Thus, this parameter should be renamed to `userDatasetId` to better reflect this limitation. We could also rename the method itself to be `unshareUserDataset`, but I suspect we'll be able to batch together multiple unshare requests in a future iteration so I propose we only change the parameter names in the interim.